### PR TITLE
QUEST LV 100: Lines 3820 - 4000

### PR DESCRIPTION
**Pull request by ttgmichael**
Will work up to Mirtis on this PR. Let me know if I missed anything.

Lines up to 4000 completed. Let me know if there are any more changes I should make before pulling.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -3823,139 +3823,139 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
-QUEST_LV_0100_20150317_003932	Defeat the Large Gray Chupacabras
-QUEST_LV_0100_20150317_003933	Large Gray Chupacabras appear when you defeated a Chupacabra. Defeat the Large Gray Chupacabras.
-QUEST_LV_0100_20150317_003934	Report back to the Supply Officer
-QUEST_LV_0100_20150317_003935	Report to Supply Officer that you defeated the Large Gray Chupacabras.
-QUEST_LV_0100_20150317_003936	Defeat the Large Gray Chupacabras that appeared when you defeated a Chupacabra 
-QUEST_LV_0100_20150317_003937	Talk to Supply Officer
-QUEST_LV_0100_20150317_003938	Seems like the Supply Officer is in trouble. Try to talk to him.
+QUEST_LV_0100_20150317_003853	$Find Search Scout in the marked area
+QUEST_LV_0100_20150317_003854	$You were asked to inform the troops to assemble before heading to Klaipeda. Inform the Search Scout about the assembly.
+QUEST_LV_0100_20150317_003855	Inform the Search Scout about the shift.
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: Forgot to add a $ here.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -3823,139 +3823,139 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003943	$Talk to Aras
+QUEST_LV_0100_20150317_003944	$Talk about what to do next in order to enter the Crystal Mine
+QUEST_LV_0100_20150317_003945	$Collect clues from Popolions
+QUEST_LV_0100_20150317_003946	$In order for Aras to permit you to enter the Crystal Mine, defeat Popolions and find some clues why the number of monsters has increased.
+QUEST_LV_0100_20150317_003947	$Give the Piece of Vubbe Cloth to the Operations Officer
+QUEST_LV_0100_20150317_003948	$Found a Piece of Vubbe Cloth. Can this be a clue? Give it to the Operations Officer
+QUEST_LV_0100_20150317_003949	$Talk to the Operations Officer
+QUEST_LV_0100_20150317_003950	$The Operations Officer seems to have realized something with the Vubbe Cloth. {nl}Try talking to him again.
+QUEST_LV_0100_20150317_003951	$Investigate the Northern Areas
+QUEST_LV_0100_20150317_003952	$The Operations Officer says there are used to be no Vubbes in the Eastern Woods. Check the northern area of the East Siauliai Woods while the Operations Officer ask Aras for help.
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: "The Operations Officer says there used to be no Vubbes in the Eastern Woods before. Check the northern area of the East Siauliai Woods while the Operations Officer asks Aras for help."

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -3823,139 +3823,139 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003869	$You seem to have taken care of Laimonas' favor. Inform the Guard Captain that you defeated the monsters on the road to Klaipeda.
+QUEST_LV_0100_20150317_003870	$Defeat Infrorocktor
+QUEST_LV_0100_20150317_003871	$Go to Klaipeda
+QUEST_LV_0100_20150317_003872	$Let's move on the way to the Klaipeda.
+QUEST_LV_0100_20150317_003873	$Defeat the Rocktortuga that suddenly appeared.
+QUEST_LV_0100_20150317_003874	$Defeat the Rocktortuga that appeared suddenly before entering the Klaipeda.
+QUEST_LV_0100_20150317_003875	$Report to the Guard Captain on Rocktortuga's defeat.
+QUEST_LV_0100_20150317_003876	$Inform the Battle Commander to return to camp
+QUEST_LV_0100_20150317_003877	$You were asked to tell troops to assemble before going to Klaipeda. Inform the Battle Commander about it.
+QUEST_LV_0100_20150317_003878	$Go to the Hanaming infested area
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: Hanaming infested area -> Hanaming-infested area

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -3823,139 +3823,139 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003879	$Go to the Hanaming infested area and quickly complete the the Battle Commander's assignment.
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: Hanaming infested area -> Hanaming-infested area

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -3823,139 +3823,139 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003949	$Talk to the Operations Officer
+QUEST_LV_0100_20150317_003950	$The Operations Officer seems to have realized something with the Vubbe Cloth. {nl}Try talking to him again.
+QUEST_LV_0100_20150317_003951	$Investigate the Northern Areas
+QUEST_LV_0100_20150317_003952	$The Operations Officer says there are used to be no Vubbes in the Eastern Woods. Check the northern area of the East Siauliai Woods while the Operations Officer ask Aras for help.
+QUEST_LV_0100_20150317_003953	$Report to the Operations Officer
+QUEST_LV_0100_20150317_003954	$Is the Vubbe Fighter controlling the Vubbes? Tell the Operations Officer about what you saw.
+QUEST_LV_0100_20150317_003955	$Defeat Vubbe Fighter's subordinates
+QUEST_LV_0100_20150317_003956	$Talk to Aras about the Vubbe Fighter.
+QUEST_LV_0100_20150317_003957	$Search for the Vubbe Fighter
+QUEST_LV_0100_20150317_003958	Is this one of the promises for the Crystal Mine? Check how many Vubbe Fighters there are.
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: Add a $ here as well.

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -3823,139 +3823,139 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003880	$Chase the Hanamings
+QUEST_LV_0100_20150317_003881	$Defeat the surrounding Hanamings
+QUEST_LV_0100_20150317_003882	$Go to the Hanaming habitat
+QUEST_LV_0100_20150317_003883	$Go to the Hanaming habitat and quickly complete the Battle Commander's assignment.
+QUEST_LV_0100_20150317_003884	$Collect Hanaming Petals to investigate.
+QUEST_LV_0100_20150317_003885	$Collect petals from Hanamings to complete the Battle Commander's assignment.
+QUEST_LV_0100_20150317_003886	$Give the Hanaming Petals to the Battle Commander
+QUEST_LV_0100_20150317_003887	$Return to the Battle Commander and give him the Hanaming Petals, so you can be relieved to go to Klaipeda.
+QUEST_LV_0100_20150317_003888	$Collect %s to investigate
+QUEST_LV_0100_20150317_003889	$Talk to the Battle Commander. Something dubious seems to have happened.
```
Comment (Grillo-del-mal) [inline QUEST_LV_0100.tsv]: "...seem to have happened"

File: UI.tsv [hunk 7]
```diff
@@ -1777,7 +1777,7 @@ UI_20150918_001776	Effect
 UI_20150918_001777	Camp duration is increased when renewed{nl} Requires %s silver.
 UI_20150918_001778	Personal Warehouse
 UI_20150918_001779	{@st41b}Type : {@sti7}%s{/}
-UI_20150918_001780	{@st41b}Carve Difficulty : {@sti7}%s{/}
+UI_20150918_001780	{@st41b}Difficulty Curve : {@sti7}%s{/}
```
Comment (Sunaries) [inline UI.tsv]: I believe this is "Class Difficulty". But, IMO it should just be "Difficulty".

Comment (ttgmichael) [inline UI.tsv]: :+1:

**Comment by ttgmichael** [comment]
Good to merge

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -3823,183 +3823,183 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
-QUEST_LV_0100_20150317_003999	Something is sealed in the crystal pillars. Check out the pillars.
-QUEST_LV_0100_20150317_004000	Defeat Demon Mirtis
-QUEST_LV_0100_20150317_004001	There were demons in the Closed Area. Let's defeat Mirtis.
-QUEST_LV_0100_20150317_004002	Defeat Mirtis
+QUEST_LV_0100_20150317_003979	$Notify the Search Scout that the Vubbe Fighter has been defeated
+QUEST_LV_0100_20150317_003980	$Return to the Search Scout and let him know that the Vubbe Fighter is defeated.
+QUEST_LV_0100_20150317_003981	$Defeat the Vubbe Fighter
+QUEST_LV_0100_20150317_003982	$Return and report to Aras 
+QUEST_LV_0100_20150317_003983	$Return to Aras and tell him that the Vubbe Fighter was defeated.
+QUEST_LV_0100_20150317_003984	$Discuss with Aras about Miners' Village
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: extra space after Miner's Village

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -3823,183 +3823,183 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003967	$Go back and give the Weaver claws to the Supply Soldier.
+QUEST_LV_0100_20150317_003968	$Defeat Weavers and collect %s
+QUEST_LV_0100_20150317_003969	$Something is not resolved yet. Try talking to the Supply Soldier.
+QUEST_LV_0100_20150317_003970	$Defeat Pokubus
+QUEST_LV_0100_20150317_003971	$Defeat the Pokubus interfering the Supply Officer.
+QUEST_LV_0100_20150317_003972	$Talk to the Supply Soldier
+QUEST_LV_0100_20150317_003973	$Tell the Supply Soldier that you defeated the Pokubus
+QUEST_LV_0100_20150317_003974	$Defeat Pokubus
+QUEST_LV_0100_20150317_003975	$Go find the Search Scout
+QUEST_LV_0100_20150317_003976	$The Operations Officer said that the Search Scout in lower area found the Vubbe Fighter. Meet the Search Scout near the bridge in the lower area of Bulvjes Farm.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: in lower area -> in the lower area

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -3823,183 +3823,183 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003983	$Return to Aras and tell him that the Vubbe Fighter was defeated.
+QUEST_LV_0100_20150317_003984	$Discuss with Aras about Miners' Village  
+QUEST_LV_0100_20150317_003985	$As soon as you thought you could finally get permission to enter the Miners' Village, the Vubbes attacked. Let's defeat them first. 
+QUEST_LV_0100_20150317_003986	$Talk to Aras again
+QUEST_LV_0100_20150317_003987	$Defeated the attacking Vubbes in a close fight. Return to Aras and get permission to enter the Miners' Village.
+QUEST_LV_0100_20150317_003988	$Defeat the monsters that are chasing the refugees
+QUEST_LV_0100_20150317_003989	$Talk to Vaidotas
+QUEST_LV_0100_20150317_003990	$You arrived at Crystal Mine 3F, where the villagers may be imprisoned. Talk to Vaidotas about finding the residents.
+QUEST_LV_0100_20150317_003991	$Search for the villagers
+QUEST_LV_0100_20150317_003992	$Vaidotas says the villagers are trapped somewhere in the 3rd Gallery. {nl}Try to look for them.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: in the 3rd Gallery -> on the 3rd floor

File: QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -3823,183 +3823,183 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
-QUEST_LV_0100_20150317_003977	Search the Vubbe Fighter
+QUEST_LV_0100_20150317_003942	$Defeat Weaver
+QUEST_LV_0100_20150317_003943	$Talk to Aras
+QUEST_LV_0100_20150317_003944	$Talk about what to do next in order to enter the Crystal Mine
+QUEST_LV_0100_20150317_003945	$Collect clues from Popolions
+QUEST_LV_0100_20150317_003946	$In order for Aras to permit you to enter the Crystal Mine, defeat Popolions and find some clues why the number of monsters has increased.
+QUEST_LV_0100_20150317_003947	$Give the Piece of Vubbe Cloth to the Operations Officer
+QUEST_LV_0100_20150317_003948	$Found a Piece of Vubbe Cloth. Can this be a clue? Give it to the Operations Officer
+QUEST_LV_0100_20150317_003949	$Talk to the Operations Officer
+QUEST_LV_0100_20150317_003950	$The Operations Officer seems to have realized something with the Vubbe Cloth. {nl}Try talking to him again.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Try talking to him again -> Talk to him again.

File: QUEST_LV_0100.tsv [hunk 12]
```diff
@@ -3823,183 +3823,183 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003929	$Defeat the Chupacabras in the supplies camp
+QUEST_LV_0100_20150317_003930	$Obtain information about Chupacabras from the Supply Officer
+QUEST_LV_0100_20150317_003931	$Listen to the information about Chupacabras from the Supply Officer.
+QUEST_LV_0100_20150317_003932	$Defeat the Large Gray Chupacabras
+QUEST_LV_0100_20150317_003933	$Large Gray Chupacabras appeared after you defeated a Chupacabra. Defeat the Large Gray Chupacabras.
+QUEST_LV_0100_20150317_003934	$Report back to the Supply Officer
+QUEST_LV_0100_20150317_003935	$Report to the Supply Officer that you defeated the Large Gray Chupacabras.
+QUEST_LV_0100_20150317_003936	$Defeat the Large Gray Chupacabras that appeared when you defeated a Chupacabra 
+QUEST_LV_0100_20150317_003937	$Talk to the Supply Officer
+QUEST_LV_0100_20150317_003938	$Seems like the Supply Officer is in trouble. Try to talk to him.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Try to talk to him. -> Talk to him.

File: QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -3823,183 +3823,183 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
...
+QUEST_LV_0100_20150317_003878	$Go to the Hanaming-infested area
+QUEST_LV_0100_20150317_003879	$Go to the Hanaming-infested area and quickly complete the the Battle Commander's assignment.
+QUEST_LV_0100_20150317_003880	$Chase the Hanamings
+QUEST_LV_0100_20150317_003881	$Defeat the surrounding Hanamings
+QUEST_LV_0100_20150317_003882	$Go to the Hanaming habitat
+QUEST_LV_0100_20150317_003883	$Go to the Hanaming habitat and quickly complete the Battle Commander's assignment.
+QUEST_LV_0100_20150317_003884	$Collect Hanaming Petals to investigate.
+QUEST_LV_0100_20150317_003885	$Collect petals from Hanamings to complete the Battle Commander's assignment.
+QUEST_LV_0100_20150317_003886	$Give the Hanaming Petals to the Battle Commander
+QUEST_LV_0100_20150317_003887	$Return to the Battle Commander and give him the Hanaming Petals, so you can be relieved to go to Klaipeda.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: It is possible to shorten "to go to Klaipeda." by replacing it with "to enter Klaipeda."

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**